lib/assets/javascripts/_modules/search.js

```diff
@@ -51,7 +53,7 @@
     this.downloadSearchIndex = function downloadSearchIndex () {
       updateTitle('Loading search results')
       $.ajax({
-        url: searchIndexPath,
+        url: pathToSiteRoot + (searchIndexPath.startsWith('/') ? searchIndexPath.slice(1) : searchIndexPath),
```

Comment:
Unfortunately this means we're getting `http_prefix/http_prefix` here as well.

I'd suggest we remove some of the complexity here and go back to just hardcoding `search.json`. Then we don't need both `data-search-index-path` and `data-path-to-site-root`.

```suggestion
        url: pathToSiteRoot + (searchIndexPath.startsWith('/') ? searchIndexPath.slice(1) : searchIndexPath),
```

Reply:
Aah that's a shame. Happy to revert to a hardcoded `search.json`, however I should note we'll be undoing the changes made in https://github.com/alphagov/tech-docs-gem/commit/3c629e753b480b85c4618ec850c32a42286c2e5a. Is that ok?

Reply:
Yep, fine by me.